concordium-contracts-common-derive/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "concordium-contracts-common-derive"
+authors = ["Concordium <developers@concordium.com>"]
+version = "0.1.0"
+edition = "2018"
+license = "MPL-2.0"
+description = "Procedural macros to ease writing for smart contracts on the Concordium blockchain."
+readme = "./README.md"
+
+[features]
+sdk = []
```

Comment:
I do not see any documentation of this feature anywhere. 

Please add it to the readme and explain why it is needed, as well as why this limited scope is chosen.

Reply:
I have updated the readme